### PR TITLE
Switch to Update Manager from GitHub Manager

### PR DESCRIPTION
**Pull request by timbocode**
Add codepotent's Update Manager.

First beta release.

### CODE REVIEW

File: vendor/composer/autoload_classmap.php [hunk 1]
```diff
@@ -265,6 +265,7 @@
     'Classic_SEO\\Traits\\Hooker' => $baseDir . '/includes/traits/class-hooker.php',
     'Classic_SEO\\Traits\\Meta' => $baseDir . '/includes/traits/class-meta.php',
     'Classic_SEO\\Traits\\Shortcode' => $baseDir . '/includes/traits/class-shortcode.php',
+    'Classic_SEO\\UpdateClient\\UpdateClient' => $baseDir . '/includes/class-update-client.php',
```
Comment (nylen) [inline vendor/composer/autoload_classmap.php]: Won't this be overwritten by `composer update`?

File: vendor/composer/autoload_static.php [hunk 2]
```diff
@@ -271,6 +271,7 @@ class ComposerStaticInitc2822413d75ee1db8fcd50de895c43fa
         'Classic_SEO\\Traits\\Hooker' => __DIR__ . '/../..' . '/includes/traits/class-hooker.php',
         'Classic_SEO\\Traits\\Meta' => __DIR__ . '/../..' . '/includes/traits/class-meta.php',
         'Classic_SEO\\Traits\\Shortcode' => __DIR__ . '/../..' . '/includes/traits/class-shortcode.php',
+        'Classic_SEO\\UpdateClient\\UpdateClient' => __DIR__ . '/../..' . '/includes/class-update-client.php',
```
Comment (nylen) [inline vendor/composer/autoload_static.php]: Won't this be overwritten by `composer update`?

**Comment by nylen** [comment]
I'm not sure how to review this. Conceptually no problem, but usually the `vendor` dir managed by `composer` is not added into source control. Any other specific input you were looking for?

File: vendor/composer/autoload_static.php (hunk 2 above)
Comment (timbocode) [reply vendor/composer/autoload_static.php]: In this case, no it doesn't. I've not included it as a dependency.

**Comment by timbocode** [comment]
Nothing specific. It's just I am moving to beta1 so just wanted a second pair of eyes. 

But the point you make about the `vendor` dir is a good one and one I'd been meaning to ask about. I notice there's no `vendor` dir in CC so where does this normally live?

As for excluding from version control, I presume this is just a case of adding to `.gitignore`?

**Pull request merged**